src/frontend/src/pages/TeamsPage/TeamMembersPageBlock.tsx

```diff
@@ -3,122 +3,168 @@
+      <Grid item>
+        <Typography sx={{ fontWeight: 'bold' }} display="inline">
+          Head:
+        </Typography>
+      </Grid>
+      <Grid item xs={true} mt={-1}>
+        <NERAutocomplete
+          id="head-autocomplete"
+          options={headOptions}
+          onChange={(_event, newValue) => newValue && setHead(newValue)}
+          filterSelectedOptions
+          size="small"
+          placeholder="Select a User"
+          value={head}
+        />
+      </Grid>
+      <Grid container direction={'row'} xs={4} md={2} lg={1}>
+        <Grid item>
+          <IconButton children={<Save />} onClick={submitHead} />
+        </Grid>
+        <Grid item>
+          <IconButton children={<Cancel />} onClick={() => setIsEditingHead(false)} />
         </Grid>
-        <Grid item xs={12}>
+      </Grid>
+    </Grid>
+  );
+
+  const EditingMembersView = () => (
+    <Grid container direction={'column'}>
+      <Typography sx={{ fontWeight: 'bold' }} display="inline">
+        Members:
+      </Typography>
+      <Grid container direction={'row'}>
+        <Grid item xs={8} md={10} lg={11}>
           <Autocomplete
             isOptionEqualToValue={(option, value) => option.id === value.id}
             filterSelectedOptions
             multiple
             id="tags-standard"
-            options={options}
+            options={memberOptions}
             value={members}
             onChange={(_event, newValue) => setMembers(newValue)}
             getOptionLabel={(option) => option.label}
-            renderInput={(params) => (
-              <TextField {...params} variant="standard" label="Members" placeholder="Select A User" />
-            )}
+            renderInput={(params) => <TextField {...params} variant="standard" placeholder="Select A User" />}
           />
         </Grid>
+        <Grid container xs={4} md={2} lg={1}>
+          <Grid item>
+            <IconButton children={<Save />} onClick={submitMembers} />
+          </Grid>
+          <Grid item>
+            <IconButton children={<Cancel />} onClick={() => setIsEditingMembers(false)} />
+          </Grid>
+        </Grid>
       </Grid>
-    </PageBlock>
+    </Grid>
   );
 
-  const nonEditingView = (
-    <PageBlock
-      title={'People'}
-      headerRight={hasPerms ? <IconButton onClick={() => setIsEditingMembers(true)} children={<Edit />} /> : null}
-    >
-      <Grid container spacing={1}>
-        <Grid item xs={12}>
-          <DetailDisplay label="Head" content={fullNamePipe(team.head)} />
-        </Grid>
-        <Grid item xs={12}>
-          <DetailDisplay label="Leads" content={team.leads.map((lead) => fullNamePipe(lead)).join(', ')} />
-        </Grid>
-        <Grid item xs={12}>
-          <DetailDisplay label="Members" content={team.members.map((member) => fullNamePipe(member)).join(', ')} />
-        </Grid>
+  const NonEditingHeadView = () => (
+    <Grid container flexDirection={'row'}>
```

Comment:
you don't need flexDirection here, Grid automatically organizes the Grid items horizontally